powersimdata/input/tests/test_transform_profile.py

```diff
@@ -85,9 +88,16 @@ def get_change_table_for_new_plant_addition(base_grid, resource):
     return ct.ct
 
 
-def _check_plants_are_scaled(ct, base_grid, base_profile_resource, resource):
-    profile_info = base_profile_resource[0]
-    base_profile = base_profile_resource[1]
+def _check_plants_are_scaled(ct, base_grid, profile_info, raw_profile, resource):
+    plant_id_type = (
+        base_grid.plant.isin(profile_type[resource]).query("type == True").index
+    )
+
+    plant_id_type = get_plant_with_resource(base_grid, resource)
```

Comment:
Does l.96 replace l. 92-94?

Reply:
Yes. I forgot to delete the first assignment

Reply:
Done